api.md

```diff
@@ -1070,6 +1156,66 @@ to the server's timezone setting.
  multi-day calls to time_bucket. The old behavior can be reproduced by passing
  2000-01-01 as the origin parameter to time_bucket.
 
+## time_bucket_gapfill() [](time_bucket_gapfill)
+
+>:TIP: TIMESTAMPTZ arguments are
+bucketed by the time at UTC. So the alignment of buckets is
+on UTC time. One consequence of this is that daily buckets are
+aligned to midnight UTC, not local time.
+>
+
+#### Required Arguments [](time_bucket_gapfill-required-arguments)
+
+|Name|Description|
+|---|---|
+| `bucket_width` | A PostgreSQL time interval for how long each bucket is (interval) |
+| `time` | The timestamp to bucket (timestamp/timestamptz/date)|
+| `start` | The start of the gapfill period (timestamp/timestamptz/date)|
+| `end` | The end of the gapfill period (timestamp/timestamptz/date)|
+
+#### Required Arguments [](time_bucket_gapfill-integer-required-arguments)
```

Comment:
I think the way that we dealt with this previously was to have a separate "units" section.  https://docs.timescale.com/v1.0/api#create_hypertable is an example (actually the only example currently)

Reply:
Hmm the supported datatypes are identical to 2 argument time_bucket and the 3rd and 4th argument is the same datatype as the 2nd.

Reply:
The one thing I would say needs to be added to match with `time_bucket` is a `### For Integer Time Inputs` before the second set of required arguments, otherwise it's not clear why there are two sets of required arguments.